autokeras/blocks/basic.py

```diff
@@ -154,6 +154,69 @@ def build(self, hp, inputs=None):
         return output_node
 
 
+class AttentionBlock(block_module.Block):
+    """The Attention Block. Luong's Multiplicative Style Attention.
```

Comment:
This description needs to add the AddtiveAttention layer.
Just change this line to "Using Attention or AdditiveAttention layer in Keras."